src/status_im2/contexts/profile/create/events.cljs

```diff
@@ -14,10 +14,11 @@
 
 (rf/defn create-profile-and-login
   {:events [:profile.create/create-and-login]}
-  [_ {:keys [display-name password image-path color]}]
+  [_ {:keys [display-name password image-path color emoji]}]
   {::create-profile-and-login
    (assoc (profile.config/create)
           :displayName        display-name
           :password           (native-module/sha3 (security/safe-unmask-data password))
           :imagePath          (profile.config/strip-file-prefix image-path)
-          :customizationColor color)})
+          :customizationColor color
+          :emoji              emoji)})
```

Comment:
just out of curiosity, why not just call `(emoji-picker.utils/random-emoji)` here?

Reply:
@J-Son89 we probably want to dispatch first into `:onboarding-2/profile-data-set`

Reply:
@J-Son89 or maybe that's useless :)

Reply:
I think we can call `(emoji-picker.utils/random-emoji)` here since that emoji is for wallet and is not related to the Status account